@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/tsd/fn_tsdWPTHandleControl.sqf

```diff
@@ -110,6 +110,9 @@ switch (_variant) do {
             case "r3": {
                 _persistState set ["ctr", 1 - (_persistState get "ctr")];
             };
+            case "t4": {
+                [_heli, _mpdIndex, "abr", createHashMapFromArray [["tsdState", _state]]] call fza_mpd_fnc_setCurrentPage;
```

Comment:
Do all subpages have it? In that case we could just add it at the root level rather than putting the same code in each